flamedisx/utils.py

```diff
@@ -294,3 +294,12 @@ def filter_kwargs(func, kwargs):
         # if func accepts wildcard kwargs, return all
         return kwargs
     return {k: v for k, v in kwargs.items() if k in params}
+
+
+@export
+def log10(x):
```

Comment:
We already have this in utils.py, it's called `tf_log10`. So this one isn't needed.